spec/models/classification_spec.rb

```diff
@@ -5,6 +5,7 @@
   it { should belong_to(:topic) }
 
   context "validations" do
+    subject { Classification.new(classifiable_id: 1, classifiable_type: Topic) }
```

Comment:
I was getting a failure I could not understand after applying this upgrade:
```
Randomized with seed 16427
F

Failures:

 1) Classification validations is expected to validate that :topic_id
    is case-sensitively unique within the scope of :classifiable_id and
:classifiable_type
    Failure/Error: it { should
validate_uniqueness_of(:topic_id).scoped_to([:classifiable_id,
:classifiable_type]) }

    NameError:
      wrong constant name dummy value

            Object.const_get(camel_cased_word)
                  ^^^^^^^^^^
    # ./spec/models/classification_spec.rb:8:in `block (3 levels) in
<main>'

Finished in 0.10979 seconds (files took 1.81 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/models/classification_spec.rb:8 # Classification
validations is expected to validate that :topic_id is case-sensitively
unique within the scope of :classifiable_id and :classifiable_type

Randomized with seed 16427
```

Found a solution [here](https://github.com/thoughtbot/shoulda-matchers/issues/1052).

Would appreciate thoughts on how to make this clearer since the `subject` appears to come out of nowhere 🙂.